libs/core/src/integration/chain-event.utils.ts

```diff
@@ -0,0 +1,446 @@
+import { EventNames, EventPairs } from '@hicommonwealth/core';
```

Comment:
another quick reminder ... at some point we should move the schemas to libs/schemas and all the integration stuff to a new lib (integrations framework TBD).

Reply:
What schemas are not in `libs/schemas`?

Reply:
I see a few under /libs/core/src/integration ... no need to move them to "shared" schemas since this is backend integration only... just tagging this folder for framework review -> in principle "core" should be about reusable frameworks without any knowledge of application level schemas or models